hyperspy/docstrings/signal.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""Common docstring snippets for signal.
+
+"""
+
+ONE_AXIS_PARAMETER = \
+    """: {int | string | axis}
+            The axis can be passed directly, or specified using the index of the
+            axis in `axes_manager` or the axis name."""
+MANY_AXIS_PARAMETER = \
+    """: {int | string | axis | tuple}
+            Either one on its own, or many axes in a tuple can be passed. In
```

Comment:
You think I should add it in this PR? Here I only expanded API without changing behaviour, as that was not the intended purpose of the PR.